src/ontology/mondo-edit.obo

```diff
@@ -178460,7 +178460,7 @@ synonym: "idiopathic purpura" RELATED [DOID:8924]
 synonym: "idiopathic thrombocytopenia" EXACT [NCIT:C3446]
 synonym: "idiopathic thrombocytopenia purpura" EXACT [NCIT:C3446]
 synonym: "idiopathic thrombocytopenic purpura" EXACT [DOID:8924, OMIM:188030]
-synonym: "immune thrombocytopenia" EXACT [Orphanet:3002]
+synonym: "immune thrombocytopenia" BROAD [Orphanet:3002]
```

Comment:
This change is actually on term MONDO:0008558 'autoimmune thrombocytopenic purpura'